playbooks/roles/postgres/tasks/main.yml

```diff
@@ -39,7 +39,7 @@
     src: "{{ POSTGRES_OPENSTACK_DB_DEVICE }}"
     fstype: ext4
     state: mounted
-  when: POSTGRES_OPENSTACK_DB_DEVICE
+  when: POSTGRES_OPENSTACK_DB_DEVICE != None
```

Comment:
Nit: I see that in other places in this repo we use `... is not none` instead of `... != None`. It might be worth changing to that for consistency.

Reply:
How about a follow up ticket to change all places? I would like to keep the context of the related ticket.

Reply:
Sure, it's not important, pure cosmetics.